_posts/2023-01-05-exploring-process-automation-opportunities.md

```diff
@@ -0,0 +1,60 @@
+
+## RPA
+
+Robotic Process Automation, or RPA, has nothing to do with mechanical robots used in different factories, but instead is a purely software-based solution, designed to mimic different tasks performed by human employees. Developing an RPA solution often starts with determining all the steps a human worker performs in a certain process, including every input and click of a button. All the potential exceptions and errors should also be documented. After scripting all the steps, the software robot can mimic the described process.
+
+The reason RPA solutions can be more productive than human workers in certain processes it that RPA solutions are faster, and they do not make any humane errors, like typos or misinterpretations. From developers’ point of view, creating RPA solutions can be fast and low effort since the solution is implementable to use existing systems without the necessity to alter said systems in any way. The RPA solution can do all the same tasks as a human worker, so access to required systems can be obtained in a comparable manner, as with human employee.
+
+The necessity of structured data and specific types of processes is the main shortcoming of RPA solutions. If the process has multiple exceptions and requires an elevated level of deduction, creating a reliable RPA solution can be next to impossible. Also, as the solutions require the processes to remain as they are, system updates and changes to the process workflow are likely to cause the solution to malfunction.
+
+## Low-code in process automation
+
+Low-code is a programming technique derived from the concepts of Rapid Application Development, where the programmer has the freedom to spend more time designing the functionality and aesthetics of a application, rather than the syntax of the code itself. Low-Code Development Platforms are based on graphical user interfaces, which allow for the users to create applications and automated workflows without writing any code. Low-code contributes to the universal shortage of professional software developers by lowering the entry barrier to create solutions and applications. This research focused on Microsoft Power Platform as the selected low-code development platform.
+
+Developing applications and automated flows with Power Platform, or any other LCDP, is faster and easier than traditional software development. Also, the simplified nature of development allows for the employees outside the IT-department to participate in application development. This makes it easier for employees who have more knowledge about the business processes themselves to make sure that the application, or automated solution, meets all the business requirements. 
```

Comment:
is there a research link for the "faster and easier" claims? Or is it part of your thesis findings.

Reply:
The claim about the LCDP development being faster and easier is a simplification of a few different research articles stating how LCDP's don't require as much technical understanding due to their GUI and this lowers the entry barrier for application development. Another sources also state how the platforms are using different components as building blocks for applications/automations and these blocks can be rapidly added, removed or modified. I didn't add the research links to the post, but the more thorough explanation (and the research citations) can be found from the thesis work itself, which is linked at the end.